dotCMS/src/main/webapp/html/common/top_inc.jsp

```diff
@@ -45,7 +45,7 @@ THIS FILE AND ITS INCLUDES
 
 	<%
 		String isNewBinaryFieldEnabled = Config.getStringProperty("FEATURE_FLAG_NEW_BINARY_FIELD");
```

Comment:
Actually, this method should be returning `true` when `FEATURE_FLAG_NEW_BINARY_FIELD` is not present, so is a Java issue, we should not do all this logic in the "frontend".

cc: @nollymar @dsilvam

Reply:
yeap, if null --> true. All this logic should be in an utility method that receives the FF name